projects/plugins/boost/app/assets/src/js/stores/critical-css-status.ts

```diff
@@ -169,6 +188,23 @@ export function storeGenerateError( error: Error ): void {
 	} ) );
 }
 
+export function resetCloudStatus(): void {
```

Comment:
Should this too be a generic name that would represent both? The only different thing here I notice is `status: 'requesting'`. Which shouldn't be a problem.